persist-cli/src/main/java/io/ballerina/persist/cmd/Init.java

```diff
@@ -205,60 +182,20 @@ private void generateSchemaBalFile(Path persistPath, String packageName) throws
         }
     }
 
-    private void updateBallerinaToml(List<String> schemas) throws BalException {
+    private void updateBallerinaToml(List<String> schemas, String module, String datastore) throws BalException {
```

Comment:
do we need schemas list here?

Reply:
removed in [566e36d](https://github.com/ballerina-platform/persist-tools/pull/148/commits/566e36dd84d453026fbf2506266f34668c08e1af)